### ISSUE TITLE
Click behaviour

### ISSUE THREAD

**Issue by edupenabad** [jonblower/hazards-lab#6]
- We should define what type of interaction we want when we clic in a storm track. Some ideas has arisen: select the track and show related information (swath, background: GDP, losses, ...); select and aggregate tracks to calculate composite statistical summaries, etc

**Comment by jonblower** [comment]
Yes, and perhaps we could allow storms to be selected by name, in a drop-down menu perhaps. It could be difficult to select a storm by point and click on a crowded map.

Could allow people to select multiple tracks with Ctrl-click, or rubber-band.
